src/crimson/os/seastore/lba_manager/btree/lba_btree.h

```diff
@@ -118,7 +118,7 @@ class LBABtree {
 
       auto get_iter() {
 	assert(pos != INVALID);
-	assert(pos < node->get_size());
+	assert(pos <= node->get_size());
```

Comment:
it can happen when splitting a lba internal node: https://github.com/ceph/ceph/blob/37fd5f486f2b84e6839a8b501bc90b516d853eec/src/crimson/os/seastore/lba_manager/btree/lba_btree.cc#L545

Actually, this commit is not enough, I'm still trying to fix this issue

Reply:
Hmm, yes, and in the next call to `split_level()`, `parent_pos.get_iter()` can get an end iterator. Seems some defects in the split path.

Reply:
> https://github.com/ceph/ceph/blob/37fd5f486f2b84e6839a8b501bc90b516d853eec/src/crimson/os/seastore/lba_manager/btree/lba_btree.cc#L545

We don't call get_iter() on the resulting iterator.  LBABtree::insert uses the iterator constructor directly to avoid that problem.

handle_split is specifically allowed to result in an iterator with a level pointing at end(), but only at the leaf.  This line is indeed wrong -- the behavior needs to be different for internal nodes.  I'll push a PR shortly with that fix.

Reply:
I think https://github.com/ceph/ceph/pull/43083 is the fix for this bit.